Dockerfile

```diff
@@ -49,6 +49,7 @@ ENV REPODIR="${HOME}/packages"
 ENV LOGDIR="${HOME}/logs"
 ENV SRCDIR="/src"
 ENV TZ=UTC
+ENV FORCE_LOCAL_VERSION=no
```

Comment:
Why don't you make this option `yes` as default?

Reply:
This repository is for general use not only for CI but also for release builds.
Basically, version constraints must be explicitly specified if needed. Otherwise, native package manager can't handle package upgrades.

Reply:
Got the concept. Thanks.